maestro-xcuitest-driver/src/main/kotlin/xcuitest/XCTestDriverClient.kt

```diff
@@ -156,4 +198,15 @@ class XCTestDriverClient(
             .port(port)
     }
 
+    fun cancelOngoingRequests() {
+        okHttpClient.dispatcher.runningCalls().filter { it.request().tag() == TAG_RUNNING_APP_CALL }
```

Comment:
why are we only cancelling those two? what about other requests?

Reply:
Actually, this is not needed anymore, since we pass a dummy response when everything is shutting down. Removed this now